node/src/service.rs

```diff
@@ -313,13 +351,10 @@ where
 		let transaction_pool = transaction_pool.clone();
 
 		Box::new(move |deny_unsafe, _| {
-			let deps = crate::rpc::FullDeps {
-				client: client.clone(),
-				pool: transaction_pool.clone(),
-				deny_unsafe,
-			};
+			let deps =
+				FullDeps { client: client.clone(), pool: transaction_pool.clone(), deny_unsafe };
 
-			crate::rpc::create_full_amplitude(deps).map_err(Into::into)
+			create_full_rpc(deps).map_err(Into::into)
```

Comment:
This is the occurrence of the one function that has been different between the four `start_node_impl_<...>` functions.